mix.exs

```diff
@@ -31,27 +29,33 @@ defmodule Mjml.MixProject do
   defp deps do
     [
       {:rustler, "~> 0.21"},
-      {:ex_doc, "~> 0.23", only: :dev, runtime: false}
+      {:ex_doc, ">= 0.0.0", only: :dev, runtime: false}
```

Comment:
> Since the project is stating `Elixir ~> 1.9`, shouldn't it be at least v0.22.0 as recommended in the [ex_doc Readme](https://github.com/elixir-lang/ex_doc#using-exdoc-with-mix)?

@paulgoetze We want to force it so it always gets the latest available `:ex_doc` version.

Reply:
👍 I thought sth. like `>= 0.22.0` would do that, but also make sure that it's at least the version that is working with the stated Elixir version? Anyways, it will pick up the one from `mix.lock`, so I guess it's not too important here.